fundingmanager.go

```diff
@@ -67,6 +67,11 @@ const (
 	// currently accepted on the Litecoin chain within the Lightning
 	// Protocol.
 	maxLtcFundingAmount = maxBtcFundingAmount * btcToLtcConversionRate
+
+	// minCommitFeePerKw is the smallest fee rate that we should propose
+	// for a new fee update. We'll use this as a fee floor when proposing
+	// and accepting updates.
+	minCommitFeePerKw = 253
```

Comment:
Do we need a separate value for LTC?

Reply:
Spec doesn't define any such value. LTC really doesn't have fees, so shouldn't be much of an issue for them...

Reply:
:trollface: